deployments/dockerfiles/bandit/Dockerfile

```diff
@@ -3,12 +3,11 @@
 
 FROM python:3.6-alpine
 
-RUN apk update && apk upgrade \
-	&& apk add --no-cache alpine-sdk bash openssh-client \
-	&& apk add git
-
-RUN pip3 install bandit
-
-RUN wget -O jq https://github.com/stedolan/jq/releases/download/jq-1.5/jq-linux64
-RUN chmod +x ./jq
-RUN cp jq /usr/bin
+# Installing bandit via pip
+RUN apk add --virtual .build-deps --no-cache git alpine-sdk bash openssh-client \
+    && pip3 install bandit \
+    && apk del .build-deps # remove virtual namespace after building is done 
+
+# Installing https://github.com/stedolan/jq
+RUN wget -O jq https://github.com/stedolan/jq/releases/download/jq-1.5/jq-linux64 \
+    && chmod +x ./jq && cp jq /usr/bin
```

Comment:
You can install `jq` with `apk` (https://pkgs.alpinelinux.org/package/edge/main/x86/jq).

<img width="1144" alt="Screen Shot 2019-10-02 at 15 00 17" src="https://user-images.githubusercontent.com/81854/66069254-63a7c200-e525-11e9-9ef9-e723f204c594.png">

Reply:
Awesome, @jairhenrique! Good catch!

Reply:
Great, will add that!

Reply:
After adding jq to apk we are down to 292Mb from the original dockerfile